temporal-sdk/src/main/java/io/temporal/internal/worker/PollerOptions.java

```diff
@@ -99,14 +101,24 @@ public Builder setPollBackoffCoefficient(double pollBackoffCoefficient) {
     }
 
     /**
-     * Initial delay in case of failure. If backoff coefficient is 1 then it would be the constant
-     * delay between failing polls.
+     * Initial delay in case of regular failure. If backoff coefficient is 1 then it would be the
+     * constant delay between failing polls.
      */
     public Builder setPollBackoffInitialInterval(Duration pollBackoffInitialInterval) {
       this.pollBackoffInitialInterval = pollBackoffInitialInterval;
       return this;
     }
 
+    /**
+     * Initial delay in case of failures that justify longer wait period (i.e. RESOURCE_EXHAUSTED
+     * errors). If backoff coefficient is 1 then it would be the constant delay between failing
+     * polls.
+     */
+    public Builder setPollBackoffLongInitialInterval(Duration pollBackoffLongInitialInterval) {
```

Comment:
@Spikhalskiy I'm open to name suggestions. I don't like that one, but I think it us best to avoid a name that would forever link this to "resource exhausted" failures.

Reply:
Let's remove "Poll" prefix from here completely. And from the neighbors.
"setCongestionInitialInterval" looks fine.
Also, this class is not a public API, we can do whatever we want with this specific class.